build.sbt

```diff
@@ -133,14 +128,13 @@ lazy val testing = (project in file("testing"))
       "com.compstak" %% "kafka-connect-migrate" % KafkaConnectHttp4sVersion % IntegrationTest,
       "io.circe" %% "circe-literal" % CirceVersion % IntegrationTest,
       "org.http4s" %% "http4s-async-http-client" % Http4sVersion % IntegrationTest,
-      "org.tpolecat" %% "doobie-postgres" % DoobieVersion % IntegrationTest,
-      "org.scalameta" %% "munit" % MunitVersion % IntegrationTest
```

Comment:
Oh, we need this for the E2E test!

Reply:
Didn't I just remove a redundant MUnit dependency? I thought there were two in that same list. :-)